public/app/plugins/datasource/elasticsearch/specs/datasource.jest.ts

```diff
@@ -1,32 +1,31 @@
 import _ from 'lodash';
-import { describe, beforeEach, it, expect, angularMocks } from 'test/lib/common';
 import moment from 'moment';
 import angular from 'angular';
-import helpers from 'test/specs/helpers';
 import { ElasticDatasource } from '../datasource';
+import { jestTimeSrvStub, TemplateSrvStub } from 'test/specs/helpers';
```

Comment:
First, I think that we shouldn't mix jest stubs with mocha test helpers. Second, I wonder how much of timeSrv and templateSrv are used by the elasticsearch datasource. If there are only a few methods I find it more explicit to define those in this test file since the stubs are related to the tests. There's probably individual taste here. 

If you still would like to have mocks of timeSrv and VariableSrv to be used by multiple tests I would suggest looking at [manual mocks in jest](http://jestjs.io/docs/en/manual-mocks.html) and evaluate if that may be a good way moving forward?

Reply:
Fixed, it seems like the mocking on the other data sources were not as big as Elasticsearch.